client/css/App.scss

```diff
@@ -1,9 +1,15 @@
 #app {
   font-family: Arial, Helvetica, sans-serif;
+  border: solid 2px black;
+  margin: 0 7%;
```

Comment:
Percentages are hard to work with. Let's figure out a `px` or `em` width for consistency.

Reply:
I am not that familiar with the benefits of each size unit. I thought that using percentages would account for the size of the screen automatically. I think we should have a discussion as a group to discuss a standard.